languages/javascript/exercises/concept/arrays/arrays.js

```diff
@@ -1,23 +1,79 @@
+/**
+ * @param {number[]} array
+ * @param {number} index
+ * 
+ * @returns {number[]}
+ */
 export function getItem(array, index) {
   throw new Error('Implement the getItem function');
 }
 
-export function setItem(array, index, newValue) {
+/**
+ * @param {number[]} array
+ * @param {number} index
+ * @param {number} newCard
+ * 
+ * @returns {number[]}
+ */
+export function setItem(array, index, newCard) {
   throw new Error('Implement the setItem function');
 }
 
-export function prefilledArray(value, length) {
-  throw new Error('Implement the prefilledArray function');
+/**
+ * @param {number[]} array
+ * @param {number} newCard
+ * 
+ * @returns {number[]}
+ */
+export function insertItemFromTop(array, newCard) {
+  throw new Error('Implement the insertItemFromTop function');
 }
 
+/**
+ * @param {number[]} array
+ * @param {number} index
+ * 
+ * @returns {number[]}
+ */
 export function removeItem(array, index) {
   throw new Error('Implement the removeItem function');
 }
 
+/**
+ * @param {number[]} array
+ * 
+ * @returns {number[]}
+ */
 export function removeItemFromTop(array) {
   throw new Error('Implement the removeItemFromTop function');
 }
 
-export function checkLengthOfStack(array, stackLength) {
+/**
+ * @param {number[]} array
+ * @param {number} newCard
+ * 
+ * @returns {number[]}
+ */
+export function insertItemFromBottom(array, newCard) {
+  throw new Error('Implement the insertItemFromBottom function');
+}
+
+/**
+ * @param {number[]} array
+ * 
+ * @returns {number[]}
+ */
+export function removeItemFromBottom(array) {
```

Comment:
```suggestion
export function removeItemAtBottom(array) {
```

Reply:
do i need to change the instruction.md function name too ?